### ISSUE TITLE
Add ability to account for DR in damage calculation/application

### ISSUE THREAD

**Issue by RGK13** [jalensailin/mvrpg#31]
Hey Jalen, I was just doing a bit of combat checking and just realized that some characters have defensive protection that take down the damage multiplier so when we don the damage count we need to redo the math every time. Would there be a way to change the multiplier in the combat chat card? Like say damage is (6*4+4) and I would need to take the multiplier value down to 3 because of a Health Damage Reduction power that the defender has...We would need to do the math every time and it can slow down the game a lot so if the system was able to support the change of the multiplier data in the chat card it would make it way faster in game.

**Comment by jalensailin** [comment]
Okay this one should be addressed (and then some). Clicking the damage icon in chat will calculate the damage using the rolling actor's full damage multiplier, BUT then you can select a token (or multiple) to apply the damage. It will then calculate the damage **taking into account DR** for each token. Let me know what you think, and if you dont find any issues, i can cut a release. Closing for now, thank you!

**Comment by RGK13** [comment]
Hi Jalen,
 something is still not right. I did the attack from Bulldozer to
Spider-Man and selected the Melee defence on the card but if you look at
the result it gave it a success at 17 and it would have taken at least 19
to hit him. And the TN number showing is 13 and I'm not sure where it's
getting that number from...

[image: image.png]

Richard Gagné
+1 507 6721-8954
***@***.***

**Comment by jalensailin** [comment]
currently Target Number has nothing to do with targeting in foundry. the default is based on the Challenging TN for your rank. so the 13 probably comes from rank 3. That's a separate issue (see #33). besides, this issue is about damage reduction, not TN

**Comment by RGK13** [comment]
I understand what I did. I usually use the targeting module (better target)
so when I was rolling I was targetting the enemy but my selected token was
still spiderman so it was using the data from the selected token. I tried
not using the target and all seems to work fine. Sorry about the confusion.
Di you think it would be possible to add the target date to avoid this for
players like me that use the target?
